java/pgv-artifacts/pom.xml

```diff
@@ -179,76 +179,31 @@
                                     <type>exe</type>
                                     <classifier>osx-x86_64</classifier>
                                 </artifact>
-                            </artifacts>
-                        </configuration>
-                    </execution>
-                    <execution>
-                        <id>osx-aarch_64</id>
-                        <goals>
-                            <goal>attach-artifact</goal>
-                        </goals>
-                        <configuration>
-                            <artifacts>
                                 <artifact>
                                     <file>${project.build.directory}/protoc-gen-validate-${project.version}-osx-aarch_64.exe</file>
                                     <type>exe</type>
                                     <classifier>osx-aarch_64</classifier>
                                 </artifact>
-                            </artifacts>
-                        </configuration>
-                    </execution>
-                    <execution>
-                        <id>linux_x86_32</id>
-                        <goals>
-                            <goal>attach-artifact</goal>
-                        </goals>
-                        <configuration>
-                            <artifacts>
+                                <artifact>
+                                    <file>${project.build.directory}/protoc-gen-validate-${project.version}-linux-aarch_64.exe</file>
+                                    <type>exe</type>
+                                    <classifier>linux-aarch_64</classifier>
```

Comment:
This was what was missing from previous fix - we were building the binary but in order for it to be published you have to attach it as an artifact to the build.